backend/src/utils/logging_config/enums/colors.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class ColorsEnum(Enum):
+    GREY = "\x1b[39;1m"
+    GREEN = "\x1b[32;1m"
+    CYAN = "\x1b[36;1m"
+    YELLOW = "\x1b[33;1m"
+    RED = "\x1b[31;1m"
+    BOLD_RED = "\x1b[41;1m"
+    BLUE = "\x1b[1;34m"
```

Comment:
Can we gather all enums in one folder? Something like: `src/core/enums/{enum_name}.py`

Reply:
Do you think it's a good idea to mix everything together? It’s not easier to read when these enums stay there what they refer to?

Reply:
I mean we could have this folder structure:
backend/
           core/
             enums/
                  __init__.py
                   logging_enums.py
                   theater_roles_enum.py
                   user_roles_enum.py
                   example_enum.py
Something like that. Of course it's your choice - I'm just making proposal